### PR TITLE
community refresh button class names and remove google maps links

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/73813

see:
moderntribe/events-community#206
moderntribe/the-events-calendar#581

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [create-venue-fields.php:173](https://github.com/moderntribe/the-events-calendar/blob/26c97821494fefd05e12095dcf7a74ce26023980/src/admin-views/create-venue-fields.php#L173)
:x: | There must be a space between if and the first parenthesis '(' | [create-venue-fields.php:176](https://github.com/moderntribe/the-events-calendar/blob/26c97821494fefd05e12095dcf7a74ce26023980/src/admin-views/create-venue-fields.php#L176)

> via jenkins/codesniffer-phpcs

File: src/admin-views/create-venue-fields.php [hunk 1]
```diff
@@ -164,6 +164,19 @@ class="tribe-dropdown"
...
+
+/**
+ * Only show the Google map toggles on the admin screens
+ * @since
+ * @central #73813
+ *
+ * @TODO ascertain version number
+ *
+ */
+if( ! is_admin() ) {
```
Comment (bordoni) [inline src/admin-views/create-venue-fields.php]: Missing space after `if`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$_VenueStateProvince' | [create-venue-fields.php:117](https://github.com/moderntribe/the-events-calendar/blob/a13f49e272561df0d927c1c6ee6430910e6f6f25/src/admin-views/create-venue-fields.php#L117)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$currentState' | [create-venue-fields.php:117](https://github.com/moderntribe/the-events-calendar/blob/a13f49e272561df0d927c1c6ee6430910e6f6f25/src/admin-views/create-venue-fields.php#L117)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'true' | [create-venue-fields.php:117](https://github.com/moderntribe/the-events-calendar/blob/a13f49e272561df0d927c1c6ee6430910e6f6f25/src/admin-views/create-venue-fields.php#L117)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'false' | [create-venue-fields.php:117](https://github.com/moderntribe/the-events-calendar/blob/a13f49e272561df0d927c1c6ee6430910e6f6f25/src/admin-views/create-venue-fields.php#L117)
:warning: | Comment refers to a TODO task | [create-venue-fields.php:173](https://github.com/moderntribe/the-events-calendar/blob/a13f49e272561df0d927c1c6ee6430910e6f6f25/src/admin-views/create-venue-fields.php#L173)

> via jenkins/codesniffer-phpcs

**Comment by zbtirrell** [comment]
this is ready to be reviewed again

**Comment by borkweb** [comment]
:+1:

**Pull request merged**